### PR TITLE
Fixes #550 norun flag

### LINKED ISSUE #550: Ensure that we use the --norun flag for the qualification runner

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
need to add the `--norun` flag to the qualificationrunner so that simulations are not calculated

File: R/utilities-qualification.R [hunk 1]
```diff
@@ -305,7 +305,7 @@ getAxesSettings <- function(axesSettingsFromConfigurationPlot) {
 startQualificationRunner <- function(qualificationRunnerFolder, qualificationPlanFile, outputFolder, options = NULL){
   optionalArguments <- paste0(options, collapse = ' ')
   qualificationRunner <- paste0(file.path(qualificationRunnerFolder, 'QualificationRunner.exe'))
-  arguments <- paste0(' -i "', qualificationPlanFile,  '" -o "', outputFolder, '" ', optionalArguments)
+  arguments <- paste0(' -i "', qualificationPlanFile,  '" -o "', outputFolder, '" ', '--norun ' optionalArguments)
```
Comment (Yuri05) [inline R/utilities-qualification.R:308]: comma before optionalArguments is missing

Comment (msevestre) [reply R/utilities-qualification.R:308]: ah...the one time I don't run the code...

Comment (Yuri05) [reply R/utilities-qualification.R:308]: Murphys law :)

Comment (msevestre) [reply R/utilities-qualification.R]: ok updated

**Pull request merged**